CourseChamp-Frontend/src/components/ViewReview.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div class="course-rating">
+      <div class="rating-header">
+        <h1>{{ rating }}/5</h1>
+        <p>Overall Quality Based on {{ numberOfRatings }} ratings</p>
+      </div>
+      <div class="course-code">
+        <h2>ECSE 428</h2>
+      </div>
+      <div class="review">
+        <p>{{ reviewText }}</p>
+        <div class="stars">
+          <span v-for="star in computedRating" :key="star" class="star">
+            ★
+          </span>
+        </div>
+        <p>Rating :</p>
+      </div>
+    </div>
+  </template>
+  
+  <script>
+  export default {
+    name: 'CourseRating',
+    data() {
+      return {
+        rating: 4.8,
+        numberOfRatings: 13,
+        reviewText: 'Great Course I loved it so much especially the part where we worked on our projects',
+      };
```

Comment:
is this temporary? I'm assuming so, please put a TODO comment on it so it's tagged and we don't forget

Reply:
added the comment 👍